ArduPlane/Attitude.cpp

```diff
@@ -370,15 +389,17 @@ void Plane::stabilize()
         const float elevator = pitchController.get_rate_out(nav_scripting.pitch_rate_dps, speed_scaler);
         SRV_Channels::set_output_scaled(SRV_Channel::k_aileron, aileron);
         SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, elevator);
+        float rudder = 0;
```

Comment:
I think this was a bug with nav scripting, it will leave the rudder at its last value unless yaw rate control is enabled. Now it will go to 0 rather than freezing at the last value.